src/python/py_openvino_genai.cpp

```diff
@@ -131,6 +143,11 @@ PYBIND11_MODULE(py_openvino_genai, m) {
         .def(py::init<>())
         .def("write", &StreamerBase::write, "Write is called every time new token is decoded. Returns a StreamingStatus flag to indicate whether generation should be stopped or cancelled", py::arg("token"))
         .def("end", &StreamerBase::end, "End is called at the end of generation. It can be used to flush cache if your own streamer has one");
+    
+    py::class_<TextStreamer, std::shared_ptr<TextStreamer>>(m, "TextStreamer", text_streamer_docstring)
```

Comment:
I think we need to highlight that `TextStreamerBase` is a base class

I would also create a dedicated `py_text_streamer.cpp` file for all streamers on python side

Reply:
Moved to `py_streamers.cpp` and highlighted that `StreamerBase` is a base class (we don't have TextStreamerBase).